src/app/@ansyn/map-facade/components/welcome-notification/welcome-notification.component.ts

```diff
@@ -0,0 +1,52 @@
+import { AfterViewInit, Component, ElementRef, HostBinding, Inject, OnDestroy } from '@angular/core';
```

Comment:
move welcome-notification.component.ts to core

Reply:
Done